content/integrations/kui.md

```diff
@@ -25,11 +25,15 @@ To run the KUI Terminal, use:
 ```sh
 npm start
 ```
+To install iter8, refer to the [iter8 installation guidelines](https://preliminary.iter8.tools/installation/).
```

Comment:
Refer to the install doc using relative reference: `../installation/kubernetes/`